LocalyticsXamarin/LocalyticsXamarin.Common/ILocalytics.cs

```diff
@@ -139,8 +177,12 @@ public interface ILocalytics
         void TagSearched(string queryText, string contentType, Int64? resultCount, IDictionary<string, string> attributes);
         void TagShared(string contentName, string contentId, string contentType, string methodName, IDictionary<string, string> attributes);
         void TagContentRated(string contentName, string contentId, string contentType, Int64? rating, IDictionary<string, string> attributes);
+        [Obsolete("TagCustomerRegistered with a dictionary has been deprecated, please use the variant with ILLCustomer instead")]
```

Comment:
Convention for the common interfaces were XL not LL. LL I thought was reserved for native code, the enumerations implement these.